src/app.css

```diff
@@ -0,0 +1,122 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  text-align: center;
+  padding: 20px;
+}
+  
+.new-thought-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 85%;
+  max-width: 500px;
+  border: 1px solid black;
+  box-shadow: 4px 4px black;
+  padding: 15px;
+  gap: 10px;
+  justify-content: center;
+  background-color: lightgray;
+  margin: auto;
+}
+
+h2 {
+  font-size: 16px;
+  font-weight: 400;
+  padding-left: 5px;
+}
+
+.text-area {
+  font-size: 16px;
+  width: 100%;
+  height: 100px;
+}
+
+.text-area::placeholder {
+  padding: 10px;
+}
+  
+.submit-button {
+  background-color: lightpink;
+  padding: 10px;
+  border-radius: 15px;
+  width: 50%;
+  min-width: 200px;
+  font-weight: bold;
+  margin-top: 10px;
+}
+  
+.submit-button:hover {
+  transform: scale(1.02);
+}
+
+.thoughts-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 30px;
+  max-width: 500px;
+  justify-content: center;
+  margin: auto;
+  margin-top: 20px;
+}
+
+.thought {
+  max-width: 500px;
+  width: 85%;
+  background-color: white;
+  border: 1px solid black;
+  box-shadow: 4px 4px black;
+  padding: 15px;
+  overflow-wrap: break-word;
+}
+
+.like-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 7px;
+  margin-top: 10px;
+  color: lightslategrey;
+}
+  
+.like-button {
+  background-color: lightgray;
+  padding: 10px;
+  border-radius: 50%;
+  border: none;
+}
+  
+.like-button:hover, .like-button.clicked {
+  background-color: lightpink;
+}
+
+.like-number {
+  padding-left: 5px;
+}
+
+.time {
+  margin-left: auto;
+  text-align: right;
+}
+  
+@media screen and (min-width: 667px) {
+  .new-thought-wrapper {
+    max-width: 600px;
+  }
+
+  .thought {
+    min-width: 600px;
```

Comment:
Should this be max-width, to make sure the content doesn't spill out of the wrapper?